### DISCUSSION TITLE
Add interlaced header info for GIF format

### DISCUSSION THREAD

**Discussion by jfcalvo** [libvips/libvips#3266] [Ideas]
Now that `interlace` support has been added to `gifsave` operation on libvips 8.14.0. Could it be possible to add a `interlaced`  output header to `vipsheader` as it's already done for JPEG and PNG files?.

**Comment by jcupitt** [comment]
Hi @jfcalvo,

Good idea. I doesn't look like libnsgif exposes this, we we'd need to add some API upstream. I made an enhancement issue: https://github.com/libvips/libvips/issues/3267
